app/src/main/java/com/example/suzette/schemas/Instruction.java

```diff
@@ -0,0 +1,29 @@
+package com.example.suzette;
+
+import java.util.List;
```

Comment:
Remove if not needed

Reply:
Are you referring to the use of lists? I could switch it to arrays instead

Reply:
No keep it List in the Recipe class. You just aren't using List in this class.

Reply:
Oh I see this is in instruction now, thanks for the catch!